httpapi/v2/handler.go

```diff
@@ -21,4 +21,7 @@ func Register(ctx map[string]interface{}, r chi.Router) {
 	r.Post("/documents", h.CreateDocument)
 	r.Patch("/documents/{"+coreapi.DocumentIDParam+"}", h.UpdateDocument)
 	r.Post("/documents/{"+coreapi.DocumentIDParam+"}/commit", h.Commit)
+	r.Get("/documents/{"+coreapi.DocumentIDParam+"}/pending", h.GetPendingDocument)
+	r.Get("/documents/{"+coreapi.DocumentIDParam+"}/committed", h.GetCommittedDocument)
```

Comment:
if a document fails to anchor and the `committed` state is not set in the document, can we still fetch the document that is stuck in committing? I believe the `committed` will still retrieve the committing document, right?

Reply:
yes, that will still fetch the document even if its not committed.